File: python_solutions/bee1115_quadrante.py
# Walber Araújo 16/11/2023

while True:
    x, y = (int(n) for n in input().split())
    if x > 0 and y > 0: print("primeiro")
    elif x < 0 and y > 0: print("segundo")
    elif x < 0 and y < 0: print("terceiro")
    elif x > 0 and y < 0: print("quarto")
    else: break

File: java_solutions/bee1115_quadrante.java
//Walber Araújo 16/11/2023

import java.util.Scanner;

public class bee1115_quadrante {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);
        int x, y;

        do {
            x = sc.nextInt();
            y = sc.nextInt();
            if (x > 0 && y > 0) {
                System.out.println("primeiro");
            } else if (x < 0 && y > 0) {
                System.out.println("segundo");
            }
            else if (x < 0 && y < 0) {
                System.out.println("terceiro");
            }
            else if (x > 0 && y < 0) {
                System.out.println("quarto");
            }
        } while (x != 0 && y != 0);
    }
}